app/decorators/plant_trial_submission_decorator.rb

```diff
@@ -45,7 +45,7 @@ def trait_descriptors
     tds = object.content.step02.trait_descriptor_list.try(:select, &:present?) || []
     numeric_tds = tds.select { |td| td.to_s.match /\A\d+\z/ }
     non_numeric_tds = tds - numeric_tds
-    non_numeric_tds | TraitDescriptor.where(id: numeric_tds).pluck(:descriptor_name)
+    non_numeric_tds | TraitDescriptor.includes(:trait).references(:trait).where(id: numeric_tds).select('traits.name')
```

Comment:
The name is somewhat misleading, but this method should return TD names, not records, so `pluck` not `select` should be used. Probably it would be good to change method name too.